one-ui/src/test/java/com/marklogic/hub/oneui/TestHelper.java

```diff
@@ -73,17 +74,26 @@ public TestHelper() throws IOException {
     }
 
     public void authenticateSession() {
-        createUser(dataHubDeveloperUsername,dataHubDeveloperPassword,"data-hub-developer");
         EnvironmentInfo environmentInfo = new EnvironmentInfo(mlHost, "DIGEST", 8000,"DIGEST", 8002,"DIGEST", 8010, "DIGEST", 8011);
         hubConfig.setCredentials(environmentInfo, dataHubDeveloperUsername, dataHubDeveloperPassword);
     }
 
     public void authenticateSessionAsEnvironmentManager() {
-        createUser(dataHubEnvironmentManagerUsername,dataHubEnvironmentManagerPassword,"data-hub-environment-manager");
         EnvironmentInfo environmentInfo = new EnvironmentInfo(mlHost, "DIGEST", 8000,"DIGEST", 8002,"DIGEST", 8010, "DIGEST", 8011);
         hubConfig.setCredentials(environmentInfo, dataHubEnvironmentManagerUsername, dataHubEnvironmentManagerPassword);
     }
 
+    public void authenticateSessionAsAdmin() {
+        EnvironmentInfo environmentInfo = new EnvironmentInfo(mlHost, "DIGEST", 8000,"DIGEST", 8002,"DIGEST", 8010, "DIGEST", 8011);
+        hubConfig.setCredentials(environmentInfo, adminUserName, adminPassword);
+    }
+
+    @PostConstruct
+    public void createTestUsers() {
```

Comment:
it is a good idea to add it after running test cases!

Reply:
@hao1st , I have called createTestUsers() wherever it's needed.